tests/e2e/specs/create_and_remove-existing-menu-test.js

```diff
@@ -0,0 +1,59 @@
+import { loginUser, visitAdminPage,activateTheme } from '@wordpress/e2e-test-utils';
+
+describe( 'Should create a new menu and remove existing menu', () => {
+
+    beforeEach( async () => {
+        await loginUser();
+	await activateTheme("twentytwentyone");
+    } );
+
+
+	it( 'Add a new menu', async () => {
```

Comment:
This only adds a new menu, so it will make sense to add it as a helper function instead of a test.